keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -441,7 +440,7 @@ version will prevent pods from starting.
 
   Yes, unit tests for the apiserver, kube-controller-manager and kubelet cover scenarios
   where the feature is disabled or enabled. Tests for transitions
-  between these states will be added before beta.
+  between these states were added before beta.
```

Comment:
Can you please add links?

Reply:
Done. Here's the link: https://github.com/kubernetes/kubernetes/pull/99446/